framework/tester.py

```diff
@@ -166,11 +166,15 @@ def setUp(self):
         time.sleep(0.2)
 
     def tearDown(self):
+        tf_cfg.dbg(2, "\tTeardown")
         for id in self.__clients:
+            tf_cfg.dbg(2, "\tStop client: %s" % id)
```

Comment:
All `client`, `tempesta` and `server` are derived from `Statefull` class which has it's own debug messages, but only for error cases. We can move all related `start|stop` messages there. May be it's good place to putt all debug messages telling what is going on with the process.